src/app/shared/components/workshop-card/workshop-card.component.ts

```diff
@@ -108,6 +110,25 @@ export class WorkshopCardComponent implements OnInit, OnDestroy {
     this.isFavorite = !this.isFavorite;
   }
 
+  onChange(): void {
```

Comment:
bad naming, it should be more specific, for example on onChangeWorkshopStatus

Reply:
done